src/sql/util.py

```diff
@@ -574,3 +557,13 @@ def enclose_table_with_double_quotations(table, conn):
         _table = _table.replace('"', "`")
 
     return _table
+
+
+def get_nested(data, keys, default=None):
```

Comment:
Please add a numpy style docstring. 
When are we passing the `default` parameter?

Reply:
Good point, I overbuilt it with the default. Removed this and added docstring.